t/t-multiple-remotes.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+. "$(dirname "$0")/testlib.sh"
+
+reponame="$(basename "$0" ".sh")"
+
+prepare_mrtest(){
+  MRHOME=$TRASHDIR/home/mrhome
+  rm -rf $MRHOME
+  mkdir $MRHOME
+  HOME=$MRHOME
+  git config --global user.name "Git LFS Tests"
+  git config --global user.email "git-lfs@example.com"
+}
+
+prepare_consumer() {
+  local consumer=$1
+  mkdir $consumer
+  cd $consumer
+  git init
+  git remote add mr file://$smain
+  git remote add fr file://$sfork
+  git fetch mr
+  git fetch fr
+  git lfs install
+}
+
+prepare_remote() {
+  local repodir="$1"
+  mkdir -p "$repodir"
+  cd "$repodir"
+  git init --bare
+  git config http.receivepack false
+  git lfs install
+}
```

Comment:
Is there a reason our existing function `setup_remote_repo` doesn't work here?  Why is setting `http.receivepack` necessary (a comment would be appropriate)?

Reply:
I added 2 comments to explain the new function.
`Set up remote repo that does not use the http API`
`Uses the basic stand alone transfer instead of the http API`
Additionally, I removed the `http.receivepack=false`. It was not necessary after all.